app/frontend/javascript/custom-tagged-plate/components/TagLayoutManipulations.spec.js

```diff
@@ -1,31 +1,29 @@
 // Import the component being tested
-import { shallowMount } from '@vue/test-utils'
+import { mount } from '@vue/test-utils'
 import TagLayoutManipulations from './TagLayoutManipulations.vue'
-import localVue from '@/javascript/test_support/base_vue.js'
 import {
   exampleQcableData,
   exampleTagGroupsList,
   exampleTagSetList,
 } from '@/javascript/custom-tagged-plate/testData/customTaggedPlateTestData.js'
 import { expect } from 'vitest'
 
-// Here are some Jasmine 2.0 tests, though you can
-// use any test runner / assertion library combo you prefer
 describe('TagLayoutManipulations', () => {
   const wrapperFactory = function () {
-    return shallowMount(TagLayoutManipulations, {
-      propsData: {
+    return mount(TagLayoutManipulations, {
+      props: {
         api: {},
         numberOfTags: 10,
         numberOfTargetWells: 10,
         tagsPerWell: 1,
       },
-      stubs: {
-        'lb-plate-scan': true,
-        'lb-tag-groups-lookup': true,
-        'lb-tag-offset': true,
+      global: {
+        stubs: {
+          'lb-plate-scan': true,
+          TagGroupsLookup: true,
```

Comment:
why is TagGroupsLookup now a class name rather than an lb- key?

Reply:
I am not sure why TagGroupsLookup cannot be resolved in the test using the lb-tag-groups-lookup name registration in the TagLayout mix-in, while the case is similar to TagSetOffset.

I understand 'lb-plate-scan' resolution because the component registration is local in TagLayoutManipulations.